goldmane/proto/methods.go

```diff
@@ -54,7 +83,8 @@ func (h *PolicyHit) fields() logrus.Fields {
 	}
 }
 
-func makeNamePart(h *PolicyHit) string {
+func makeNamePart(h *PolicyHit) (string, error) {
+	logrus.WithFields(h.fields()).Debug("Generating name part from policy hit")
 	var namePart string
 	switch h.Kind {
```

Comment:
For consistency in the naming scheme we've chosen for kubernetes network policy, can we define the kind to `PolicyKind_KubernetesNetworkPolicy` instead of `PolicyKind_NetworkPolicy`?

Reply:
I think we want to leave it like so - the NetworkPolicy kind is the native Kubernetes type, so should get the name :)